server/plugin.go

```diff
@@ -24,20 +24,16 @@ import (
 
 const (
 	GITHUB_TOKEN_KEY        = "_githubtoken"
-	GITHUB_STATE_KEY        = "_githubstate"
 	GITHUB_USERNAME_KEY     = "_githubusername"
 	GITHUB_PRIVATE_REPO_KEY = "_githubprivate"
 	WS_EVENT_CONNECT        = "connect"
 	WS_EVENT_DISCONNECT     = "disconnect"
 	WS_EVENT_REFRESH        = "refresh"
 	SETTING_BUTTONS_TEAM    = "team"
-	SETTING_BUTTONS_CHANNEL = "channel"
-	SETTING_BUTTONS_OFF     = "off"
```

Comment:
These are also defined in the webapp here : https://github.com/mattermost/mattermost-plugin-github/blob/f67b80b3daca4b99b036b59759148e241ac44c2a/webapp/src/constants/index.js#L6-L7

We should remove them there to be consistent.